File: Javascript/A_NivelIniciante/Exercicio_1020.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
console.log(parseInt(lines[0]/365) + " ano(s)\n" + parseInt(lines[0]%365/30) + " mes(es)\n" + lines[0]%365%30 + " dia(s)")

File: Python/A_NivelIniciante/Exercicio_1020.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
#url do problema - https://www.urionlinejudge.com.br/judge/pt/problems/view/1020
dias = int(input())
anos = dias/365
meses = (dias -(anos * 365))/30
dias = dias -((anos * 365) + (meses * 30))
print("%d ano(s)" %anos)
print("%d mes(es)" %meses)
print("%d dia(s)" %dias)
